app/controllers/api/v1/exercises_controller.rb

```diff
@@ -91,16 +91,34 @@ def index
       #{json_schema(Api::V1::Exercises::Representer, include: :writeable)}
     EOS
     def create
+      exercise = Exercise.new
       user = current_human_user
-      standard_create(
-        Exercise.new, Api::V1::Exercises::Representer, user: current_api_user
-      ) do |exercise|
-        exercise.publication.authors << Author.new(
-          publication: exercise.publication, user: user
-        ) unless exercise.publication.authors.any? { |au| au.user == user }
-        exercise.publication.copyright_holders << CopyrightHolder.new(
-          publication: exercise.publication, user: user
-        ) unless exercise.publication.copyright_holders.any? { |ch| ch.user == user }
+
+      create_options = { status: :created, location: nil }
+      represent_with_options = {
+        user_options: { user: current_api_user }, represent_with: Api::V1::Exercises::Representer
+      }
+
+      Exercise.transaction do
+        consume!(exercise, represent_with_options.dup)
+
```

Comment:
is the `.dup` needed?  I only see it being merged in below

Reply:
I was afraid `consume!` would do something unexpected to my hash, since it does modify the model (and save it) and stuff